cargo/lib/dependabot/cargo/update_checker/version_resolver.rb

```diff
@@ -188,6 +185,7 @@ def handle_cargo_errors(error)
           end
 
           if error.message.include?("authenticate when downloading repo") ||
+             # TODO stop catching this 200 error: https://github.com/dependabot/dependabot-core/pull/5332#discussion_r936888624
```

Comment:
This could be pulled into a separate PR

Reply:
good idea, I'll open a draft PR shortly... I'd also like to leave in the code comment so that we don't lose track of it...

Reply:
https://github.com/dependabot/dependabot-core/pull/5967